src/national.py

```diff
@@ -21,14 +25,64 @@
 
 logger = structlog.stdlib.get_logger()
 
-
 adjust_limit = float(os.getenv("ADJUST_MW_LIMIT", 0.0))
 get_plevels = bool(os.getenv("GET_PLEVELS", True))
 
 router = APIRouter(
     tags=["National"],
 )
 
+# Initialize Elexon API client
+api_client = ApiClient()
+forecast_api = GenerationForecastApi(api_client)
+
+
+@router.get("/elexon", summary="Get BMRS Solar Forecast")
+@limiter.limit(f"{N_CALLS_PER_HOUR}/hour")
+def get_elexon_forecast(
+    request: Request,
+    start_datetime_utc: datetime = Query(
+        default=datetime.utcnow() - timedelta(days=3), description="Start date and time in UTC"
+    ),
+    end_datetime_utc: datetime = Query(
+        default=datetime.utcnow() + timedelta(days=3), description="End date and time in UTC"
+    ),
+    process_type: str = Query("Day Ahead", description="Process type"),
+):
+    """
+    Fetch BMRS solar forecasts from the Elexon API.
+
+    Args:
+        request (Request): The request object containing metadata about the HTTP request.
+        start_datetime_utc (datetime): The start date and time in UTC.
+        end_datetime_utc (datetime): The end date and time in UTC.
+        process_type (str): The type of process (e.g., 'Day Ahead').
+
+    Returns:
+        dict: Dictionary containing the fetched BMRS solar forecast data.
+
+    External API:
+        [Elexon API Documentation]
+        (https://bmrs.elexon.co.uk/api-documentation/endpoint/forecast/generation/wind-and-solar/day-ahead)
+    """
+    response = forecast_api.forecast_generation_wind_and_solar_day_ahead_get(
```

Comment:
Worth wrapping this in a `try ... except` block to handle the errors specific to the external API not working?

Reply:
I actually asked the try to br removed. I yhink dkecon is done so rarely it adds more code thsn it's worth

Reply:
Right, okay. Not sure if that's a Python thing or a personal preference, but for the record I do _not_ agree that defensive code is not worth ~4 extra lines... instead, the user will just get an exception through, `Internal Server Error` and no real indication of what's wrong – and neither will we, so we'll clamber into the logs to find out...

Reply:
@braddf ill comment on the new PR with this